client/allocrunner/alloc_runner.go

```diff
@@ -63,30 +63,26 @@ type allocRunner struct {
 	// vaultClient is the used to manage Vault tokens
 	vaultClient vaultclient.VaultClient
 
-	// waitCh is closed when the Run() loop has exited
+	// waitCh is closed when the Run loop has exited
```

Comment:
Sorry for the linenoise. I got sick of hitting `/Run() ` in vim and having to hop through all of these comments :sweat_smile: